pom.xml

```diff
@@ -39,11 +39,10 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-
             <dependency>
                 <groupId>com.google.cloud</groupId>
                 <artifactId>libraries-bom</artifactId>
-                <version>9.1.0</version>
+                <version>20.2.0</version>
```

Comment:
thats a big jump :)

Reply:
Agreed, though it's what I started with and it seemed to work just fine.  It seems like loads, but 9.1 came out in August 2020